src/main/java/bot/UserInterface.java

```diff
@@ -0,0 +1,121 @@
+                + "\n-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~\n");
+    }
+
+    /**
+     * Shows the error message to the user, specifically when local data is not found.
+     */
+    public void showLoadingError() {
+        System.out.println("-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~\n"
+                + "Creating a new storage for the user..." + "\n-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~\n");
+    }
+
+    /**
+     * Shows the welcome message to the user.
+     */
+    public void showWelcomeMessage() {
+        System.out.println("-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~\n"
+                + "Hello! My name is Duke!\n\n"
+                + " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n\n"
+                + "What can I do for you today?\n"
+                + "-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~\n");
+    }
+
+    /**
+     * Shows the exit message to the user.
+     */
+    public void showExitMessage() {
+        System.out.println("-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~\n" + "Bye... Hope to see you again soon!\n"
+                + "-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~\n");
+        Bot.stop();
+    }
+
+    /**
+     * Shows the message for a task being added, according to the type of task given.
+     *
+     * @param input  The task to be added to the list.
+     * @param type   The type id. Todo: 1, Deadline: 2, Event: 3.
+     * @param length The current length of the list.
+     * @param time   The information regarding the task (In the format of "(By: ...)" or "(At: ...)", or "" for Todos)
+     */
+    public static void showTaskAdded(String input, int type, int length, String time) {
+
+        // Determine the string that displays the type of task
+        String taskType;
+        String prefix;
+
+        switch (type) {
+            case 1:
+                taskType = "task (Todo)";
+                prefix = "[T][ ]";
+                break;
+            case 2:
+                taskType = "task (Deadline)";
+                prefix = "[D][ ]";
+                break;
+            case 3:
+                taskType = "task (Event)";
+                prefix = "[E][ ]";
+                break;
+            default:
+                taskType = "";
+                prefix = "";
+        }
+
+        // Return the message accordingly
+        System.out.println("-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~\n" + "Alright. I've added the following "
+                + taskType + ":\n--> " + prefix + " " + input
+                + (type == 2 ? "(By: " + LocalDateTime.parse(time, inputFormatter).format(outputFormatter) + ")"
+                        : type == 3 ? "(At: " + LocalDateTime.parse(time, inputFormatter).format(outputFormatter) + ")"
+                                : "")

```

Comment:
This part seems pretty complicated, perhaps you could do this in steps like: 
```java
String taskDesc = taskType + ":\n--> " + prefix + " " + input; 

String taskTime = type == 2
        ? "(By: " + LocalDateTime.parse(time, inputFormatter).format(outputFormatter) + ")"
        : type == 3
            ? "(At: " + LocalDateTime.parse(time, inputFormatter).format(outputFormatter) + ")"
            : "";

System.out.println("-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~-~\n"
        + "Alright. I've added the following " + taskDesc + taskTime + ...) 
```
Or even place the expression for taskTime into a separate method!

Reply:
Noted - made changes. Thanks!